pertpy/tools/_milo.py

```diff
@@ -387,32 +400,35 @@ def da_nhoods(
                 from rpy2.robjects.packages import STAP
 
                 get_model_cols = STAP(r_str, "get_model_cols")
-                model_mat_cols = get_model_cols.get_model_cols(design_df, design)
-                model_df = pd.DataFrame(model)
+                with localconverter(ro.default_converter + numpy2ri.converter + pandas2ri.converter):
+                    model_mat_cols = get_model_cols.get_model_cols(design_df, design)
+                with localconverter(ro.default_converter + pandas2ri.converter + numpy2ri.converter):
```

Comment:
```suggestion
                with localconverter(ro.default_converter + pandas2ri.converter):
```

should also work here no?

Reply:
Ah I specifically remember this, we need that! Somehow it messes with `model_df` and we need to do that and then convert it to pandas dataframe